ludralph/PostIt-Raphael-Etim
# Bugfix message board

#### What does this PR do?

This Pull request fixes bug with managing redux state in creating groups and posting messages to groups.
#### Description of Task to be completed?

A user should be able to create groups, view his created groups, add members to a group. The user should also be able to post messages to created groups.

How should this be manually tested?

On account login, the user will click on the create group button and enter the name of his group. Once the modal is closed, the user should see the newly created group in in a list-group. Subsequent groups created by the user should also be visible within the list group.

On clicking on a group, a message board should be visible to the user. The user can post messages to the message board with priorities selected.

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/13855608/33837548-f9837572-de8c-11e7-832b-126653ef1f47.png)
![image](https://user-images.githubusercontent.com/13855608/33837600-2afdb504-de8d-11e7-896b-f4697e889974.png)


Questions:
